src/Components/ArticleLists/ArticlesList.jsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+import ArticleCard from "./ArticlesCard";
+import '../CSS/ArticleCard.css';
+
+
+export default function ArticlesLists() {
+    const [articles, setArticles] = useState([]);
+
+    useEffect(() => {
+        getArticles().then(({ articles }) => {
+            setArticles(articles);
+        });
+    }, []);
+
```

Comment:
I'd consider putting in some loading state to let the user know something is happening, especially for those on a slower internet connection 👀